jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT

```diff
@@ -23,8 +23,8 @@ export GPREFIX="${GDUMP}.t${gcyc}z."
 export APREFIX="${CDUMP}.t${cyc}z."
 
 # Generate COM variables from templates
-RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_HISTORY:COM_ATMOS_HISTORY_TMPL
-RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_HISTORY_PREV:COM_ATMOS_HISTORY_TMPL \
+                                                     COM_ATMOS_ANALYSIS_PREV:COM_ATMOS_ANALYSIS_TMPL
```

Comment:
not my day ... I got that wrong too. I don't need the analysis from the previous cycle.